.vscode/launch.json

```diff
@@ -9,18 +9,33 @@
             "request": "launch",
             "name": "Debug LLDB",
             "program": "node",
+            "windows": {
+                "program": "C:\\Progra~1\\nodejs\\node.exe"
```

Comment:
Will node.exe always be placed here or could it be installed elsewhere?

Reply:
This is the default install location on Windows. The problem is that by only having "program: node" VS Code finds the C:\Program Files\ path and VSCode seems to mess up and tries to run the given "program" from the "current dir\Files\nodejs\node.exe. So the build fails with cryptic message like path not found C:\realm-js\Files\nodejs\node.exe. This is a bug I guess is in VS Code where it fails to handle paths with spaces. Previous versions of VS Code were working fine with this. So here we encode the path with the shorthand 8.3 version to the default location. This way at least the fail message will be exact so the developer will know what is not found and where it was searched for. 

If you have any better suggestion will be glad to change this.